autorest/codegen/models/credential_info.py

```diff
@@ -0,0 +1,13 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from enum import Enum
+
+class CredentialInfo(str, Enum):
+    AAD_TOKEN = "AADToken"
```

Comment:
@msyyc can you update this to the new naming with the new modelerfour?

It's also confusing having it have both security information and actual info about credentials. I think this means even more we should have a model that represents "this is the information I need to generate a credential", and remove the security-specific information from the model.

Reply:
@iscai-msft , where to set modelerfour version for CI?

Reply:
in the [readme file](https://github.com/Azure/autorest.python/blob/autorestv3/README.md). You should also update the min modelerfour listed in the changelog

Reply:
done